<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createGastos</class>
 <widget class="QDialog" name="createGastos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>396</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>428</width>
    <height>356</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>428</width>
    <height>396</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Novo gasto</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/Icons/main_icon.png</normaloff>:/Icons/main_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-image: url(:/Icons/background2.jpg);
}
QLabel{
	color:#FF692E;
	font: 12pt &quot;Segoe UI&quot;;
}
QPushButton{
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #ff9b58, stop: 0.4 #ff7037,
                                stop: 0.5 #ff7037, stop: 1.0 #ff9b58);
	border:none;
	padding: 10px;
	color:#FFF;
}
QLineEdit{
	border:none;
	background-color: transparent;
	padding:5px;
	border-bottom: 1px solid #000;
}
QDateEdit{
	padding:5px;
}
QCheckBox{
	color:#FF692E;
	margin: 10px 0;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="10" column="0">
      <widget class="QLabel" name="labelCate">
       <property name="text">
        <string>Categoria:</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="labelDesc">
       <property name="text">
        <string>Descrição:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" alignment="Qt::AlignLeft">
      <widget class="QLineEdit" name="valorInput">
       <property name="placeholderText">
        <string>200.00</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0" colspan="2">
      <widget class="QTextEdit" name="descInput"/>
     </item>
     <item row="15" column="1" alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="cancelButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0" alignment="Qt::AlignRight">
      <widget class="QPushButton" name="saveButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Salvar</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QComboBox" name="categoriasComboBox">
       <property name="maxVisibleItems">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QCheckBox" name="mensalInput">
       <property name="text">
        <string>Gasto mensal</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" alignment="Qt::AlignLeft">
      <widget class="QLabel" name="labelValor">
       <property name="text">
        <string>Valor:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QDateEdit" name="dateInput">
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="date">
        <date>
         <year>2000</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelData">
       <property name="text">
        <string>Data:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1" alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="createCatGastoButton">
       <property name="styleSheet">
        <string notr="true">padding:0;
background-color:none;</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/Icons/icons8-plus.svg</normaloff>:/Icons/icons8-plus.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
